src/igr-tests/add-rm-dep/run-test.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
-../../dinit -d sd -u -p socket -q main &
+cd $(dirname $0)
```

Comment:
This should be `cd "$(dirname $0)"`. I.e. there should be quotes around any expansion that is meant to yield a single token but which comes from arbitrary path. If `dirname $0` yields a path with a shell-special character or whitespace, this won't work as is.